jsk_baxter_robot/baxtereus/baxter-interface.l

```diff
@@ -1,5 +1,6 @@
 (require "package://pr2eus_moveit/euslisp/robot-moveit.l")
 (require :baxter "package://baxtereus//baxter-util.l")
+(require :roseus_resume "package://roseus_resume/euslisp/interruption-handler.l")
```

Comment:
I think this line causes error when `roseus_resume` is not installed in workspace.
Fix this or fix every .rosinstall to include `roseus_resume`.